recipe/meta.yaml

```diff
@@ -41,14 +41,33 @@ test:
     - pandas
   commands:
     - pip check
+    # Skip test suite on PyPy as it segfaults there
+    # xref: https://github.com/conda-forge/pandas-feedstock/issues/148
+    #
+    # Also skip `test_rolling_var_numerical_issues` on `ppc64le` as it is a known test failure.
+    # xref: https://github.com/conda-forge/pandas-feedstock/issues/149
+    {% set extra_args = ["-m not clipboard and not single_cpu", "--skip-slow", "--skip-network", "--skip-db", "-n=2"] %}
+    {% set extra_args = extra_args + ["--skip-arm_slow"] %}   # [aarch64 or ppc64le]
```

Comment:
Adding [`arm_slow` marker]( https://github.com/pandas-dev/pandas/blob/v1.5.2/pyproject.toml#L61 ) for both `aarch64` and `ppc64le` to cutdown test runtime